docs/receiving/receiving-with-ingest.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Ingest Overview
+---
+
+import CodeTabs from '@theme/CodeTabs';
+import TabItem from '@theme/TabItem';
+
+[Svix Ingest] grants you scalability, security best practices, and reliability,
+all with minimal configuration.
+
+Creating a `Source` will generate an endpoint you can share with a webhook
+provider as a destination for their webhooks.
+
+Ingest supports a variety of signature verification schemes used by webhook
+providers. Supported values for a `Source`'s' `type` include:
+
+- `beehiiv`
+- `brex`
+- `clerk`
+- `github`
+- `guesty`
+- `hubspot`
+- `incidentIo`
+- `lithic`
+- `nash`
+- `pleo`
+- `replicate`
+- `resend`
+- `safebase`
+- `sardine`
+- `shopify`
+- `stripe`
+- `stych`
+- `svix` (that's us!)
+- `zoom`
+
+Additionally there's the option to configure a `Source` as `http` that doesn't
```

Comment:
I'm feeling like we should rename the `http` source type to `unverified` or something. 

It was confusing to the team when discussed during grooming, with folks assuming it had something to do with http vs https.

Reply:
I agree

Reply:
Updated this to mention `unverified` and I've got a PR up to make that a real option in the backend.

Reply:
I set this back to HTTP for the time being because we haven't yet agreed on what this should be called.

Right now, my best idea is "other"